src/pages/community/events.js

```diff
@@ -23,26 +23,27 @@ const breadcrumbs = [
     { label: 'Events', target: '/community/events' },
 ];
 
+const PAGE_DESCRIPTION =
+    'Join us in a city near you to connect with MongoDB users who are shaking up their industries, and be among the first to hear updates and exciting news.';
+
 export default () => {
     const [events, error, isLoading] = useEventData();
+    const { title, siteUtl } = useSiteMetadata();
```

Comment:
```suggestion
    const { title, siteUrl } = useSiteMetadata();
```

Looks like this is a typo